packages/app/src/contacts/components/contact-group/contact-group.styled.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
(adding it here as it is easier to have a thread)

1. Row context menu in some cases is clipped
2. There are two scrolls in the list - one for the whole list and one for the "letter group"

https://user-images.githubusercontent.com/41268731/189847637-1ccc93ba-8e62-4ee8-87e3-3d279f3e4e4f.mov